### ISSUE TITLE
[Bug]: Netherite Sluice requires RF and holds too much rf

### ISSUE THREAD

**Issue by Kevin-Marsh** [FTBTeam/FTB-Modpack-Issues#12]
### Modpack

FTB OceanBlock

### Modpack version

1.1.0

### Describe the bug

I cannot seem to run the netherite sluice without RF. From the description it seemed like it would operate without RF but be more efficient and faster with RF. It also seems to draw down and store an absurd amount of rf with mine storing 71 MFE. Highend devices from other mods that use vastly more power only store 50 KFE.

### Steps to reproduce

1. Make a netherite sluice
2. Try to use it Without RF and fail
3. Hookup Power
4. Watch it drain your base until it is full
5. Break and move the sluice for whatever reason and waste all that power.

### Expected behaviour

Have Netherite Sluice work without RF and be more efficient with RF.

**Comment by Sunekaer** [comment]
Hi,

Power storage is based on the cost of running the sluice when it's fully upgraded, therefor this value won't be changed.
The requiring of rf for the sluice to work ill talk to the mod dev about and see if we wanna change that or not.

**Comment by Kevin-Marsh** [comment]
Hi,

I hadn't actually seen the upgrade module prior to my post and I can definitely see where you are coming from with regards to the RF. I would however like to point out that the upgrades state that the formula for power consumption is 90 + (1.28 ^ upgrades). This is fairly reasonable since the power usage would increase incrementally effectively costing an extra 1.28 - 2 rf per tick per upgrade implemented. I have been looking in to it and the way that it has been setup to actually work is 90 times (1.28 ^ upgrade). This setup of times not plus is what it causing the patently absurd increase in power and is what is causing the rf issue. For example. With the current setup as it is, if I add 8 speed upgrades to a netherite sluice it then works out to be 90 times 7.20 which is 648fe. So a 40% increase in processing speed equals a seven times increase in power consumption. You could instead build 7 netherite sluices all running without an upgrade and get a 700% increase instead of a 40% increase.

In summary: someone messed up the upgrades and used times instead of plus for the upgrade formula causing the ridiculous power storage requirements and making the netherite sluice actually useless.

**Comment by Sunekaer** [comment]
At the point where you can upgrade your sluice you basically has unlimited power so this isn't an issue.
Power cost wont be changed
